javascript/ql/src/Security/CWE-020/UselessRegExpCharacterEscape.ql

```diff
@@ -0,0 +1,144 @@
+/**
+ * @name Useless regular expression character escape
```

Comment:
```suggestion
 * @name Useless regular-expression character escape
```

Reply:
How about "useless backslash escape" as a simpler alternative?

Reply:
I would like to keep "regular expression" in the name to separate this query from the dual query that flags the boring cases that do not change regexp semantics - as implemented in `UselessCharacterEscape`. We may want to productice the latter one day.

Reply:
Fair enough.